src/components/progressiveImage/progressiveImage.api.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ProgressiveImage",
+  "category": "animationsAndGestures",
```

Comment:
I think we decided to drop this category for now and put these components under basic
We'll probably do the same for our demo app

Reply:
ok